docs/customization.md

```diff
@@ -165,6 +165,43 @@ For more on this topic refer to the [MkDocs documentation][4]
 
   [4]: http://www.mkdocs.org/user-guide/styling-your-docs/#overriding-template-blocks
 
+## Extending search
```

Comment:
There are some grammatical mistakes in this documentation, e.g. articles missing etc.